packages/xo-server-sdn-controller/src/index.js

```diff
@@ -715,6 +671,70 @@ class SDNController extends EventEmitter {
     }
   }
 
+  async _addHostToPoolNetworks(host) {
+    const xapi = host.$xapi
+
+    const tunnels = filter(xapi.objects.all, { $type: 'tunnel' })
+    for (let i = 0; i < tunnels.length; ++i) {
+      const tunnel = tunnels[i]
```

Comment:
If the the index of the loop is used only for this purpose `const tunnel = tunnels[i]` than it's better to use this kind of loop:
```js
for(let tunnel of tunnels) {
...
```
And get rid of `const tunnel = tunnels[i]`